runtime/nvqir/NVQIR.cpp

```diff
@@ -359,6 +359,14 @@ Result *__quantum__qis__mz__body(Qubit *q) {
   return b ? ResultOne : ResultZero;
 }
 
+bool __quantum__qis__read_result__body(Result *result) {
+  // TODO: implement post-measurement result retrieval. This is not needed for
```

Comment:
Do we need to file an issue?

Reply:
Issue #758 written. I also added a reference to this new issue at this spot in the code (d57dbe22).